src/components/victory-axis.jsx

```diff
@@ -47,9 +47,17 @@ const styles = {
   }
 };
 
-class VAxis extends React.Component {
+@Radium
+export default class VictoryAxis extends React.Component {
+  static role = "axis";
```

Comment:
ah-ha. the child objects are wrapped in RadiumEnhancer. `child.type` has the expected behavior though. I guess our options are `child.type === VictoryAxis` or `child.type.role === axis`.  I think the former is the blessed approach, but the latter would allow folks to write their own axis implementation.